lib/config/configuration.go

```diff
@@ -912,6 +912,16 @@ func applyProxyConfig(fc *FileConfig, cfg *service.Config) error {
 		}
 		cfg.Proxy.MongoPublicAddrs = addrs
 	}
+	if len(fc.Proxy.PeerPublicAddr) != 0 {
+		if fc.Proxy.PeerAddr == "" {
+			return trace.BadParameter("peer_listen_addr must be set when peer_listen_public_addr is set")
```

Comment:
What are the consequences of PeerPublicAddr being set but PeerAddr not?

Should we copy PeerPubliAddr to PeerAddr if this is the case?